spring-boot-project/spring-boot/src/main/java/org/springframework/boot/diagnostics/analyzer/FlywayMigrationScriptMissingFailureAnalyzer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.diagnostics.analyzer;
```

Comment:
This failure analyzer should belong with the rest of the Flyway support. For insance `HikariDriverConfigurationFailureAnalyzer` is in the `jdbc` package of the auto-configuration module. Can you please move this class to the auto-configuration module as well (in the `flyway` package)?

Reply:
Sounds good. I have moved this class under flyway package of auto-configuration module.